tests/phpt/declare/07_strict_types_kphpinfercast_error.php

```diff
@@ -0,0 +1,8 @@
+@kphp_should_fail
+/pass string to argument \$s of infercast_int_param/
+/but it's declared as @param int/
+<?php
+
+require_once __DIR__ . '/infercast_func.php';
+require_once __DIR__ . '/can_call_with_cast.php'; // OK
+require_once __DIR__ . '/cant_call_with_cast.php'; // Error
```

Comment:
Shouldn't these two files have a '.inc' extenstion?

Reply:
Done. I added `inc` to all php files that don't have `@` tag and are intended to be included from other test files (that have `@` tag).